src/hostapi/wdmks/pa_win_wdmks.c

```diff
@@ -6582,31 +6582,41 @@ static PaError PaPinRenderSubmitHandler_WaveCyclic(PaProcessThreadInfo* pInfo, u
 static PaError PaPinCaptureEventHandler_WaveRTEvent(PaProcessThreadInfo* pInfo, unsigned eventIndex)
 {
     unsigned long pos;
-    unsigned realInBuf;
-    unsigned frameCount;
+    unsigned frameCount = 0;
+    unsigned bytesToRead;
     PaWinWdmIOInfo* pCapture = &pInfo->stream->capture;
     const unsigned halfInputBuffer = pCapture->hostBufferSize >> 1;
     PaWinWdmPin* pin = pCapture->pPin;
     DATAPACKET* packet = 0;
```

Comment:
Is this an unused variable? If so please remove it.

Reply:
The variable is "packet".